subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/JUnitXmlReport.java

```diff
@@ -57,4 +57,25 @@ public interface JUnitXmlReport extends DirectoryReport {
     @Input
     Property<Boolean> getMergeReruns();
 
+    /**
+      * Should omit system out log
+     */
+    void setOmitSystemOutLog(boolean omitSystemOutLog);
```

Comment:
❗ These will be new properties in a public Gradle API.

Please mark each with the `@Incubating` annotation and add a line in the javadoc with the tag `@since 8.1`.  Our test automation will fail builds without these present.

You can check this yourself by running `gradle sanityCheck`.

Reply:
done